src/__tests__/App.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import store from '../redux/configureStore';
+import Coins from '../components/Coins';
+
+const renderMissions = (
+  <Provider store={store}>
+    <Coins />
+  </Provider>
+);
+
+describe('Coins', () => {
+  test('renders coins page', () => {
+    render(renderMissions);
+    const header = screen.getByText('Coins');
+    expect(header).toBeInTheDocument();
+  });
+});
```

Comment:
- [ ] This test is currently failing. Kindly fix it so it passes 😎 
![image](https://user-images.githubusercontent.com/46839548/182954714-3cf8ba5e-05e2-4f82-960c-e08877e71cab.png)